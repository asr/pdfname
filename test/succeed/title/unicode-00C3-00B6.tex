\documentclass{article}
\usepackage{hyperref}

% Title: Unicode `Ã¶` (`U+00C3` and `U+00B6`).

\hypersetup{%
  pdfauthor={Author},%
  pdftitle={GÃ¶odel},%
  pdfcreationdate={D:2017},%
}

\begin{document}
Hello World!
\end{document}
