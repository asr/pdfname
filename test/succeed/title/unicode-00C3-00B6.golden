ret > ExitSuccess
out > The full path name will be /tmp/author-2017.godel.pdf
out >
